packages/tradershub/src/features/accounts/constants.tsx

```diff
@@ -0,0 +1,50 @@
+import IcBlurryDocument from '../../public/images/accounts/ic-blurry-document.svg';
+import IcCroppedDocument from '../../public/images/accounts/ic-cropped-document.svg';
+import IcDocumentAddressMismatch from '../../public/images/accounts/ic-document-address-mismatch.svg';
+import IcDocumentNameMismatch from '../../public/images/accounts/ic-document-name-mismatch.svg';
+import IcEnvelope from '../../public/images/accounts/ic-envelop.svg';
+import IcOldIssuedDocument from '../../public/images/accounts/ic-old-issued-document.svg';
+import { THooks } from '../../types';
+
+type TExampleImageConfig = {
+    description: React.ReactNode;
```

Comment:
```suggestion
    description:ReactNode;
```

Import ReactNode directly from react